capstone/src/main/resources/static/map.js

```diff
@@ -46,33 +70,53 @@ function createMarker(event,campusMap) {
   newMarker.addListener('click', function() {
     eventInfoWindow.open(campusMap, newMarker);
   });
+  var eventInfoWindow = createInfoWindow(event, campusMap);
```

Comment:
Where is this info window used? No info window is showing up on the form.

Reply:
should be fixed now!